src/Cli/dotnet/ParseResultExtensions.cs

```diff
@@ -13,9 +13,22 @@ namespace Microsoft.DotNet.Cli
 {
     public static class ParseResultExtensions
     {
+        ///<summary>
+        /// Finds the command of the parse result and invokes help for that command.
+        /// If no command is specified, invokes help for the application.
+        ///<summary>
+        ///<remarks>
+        /// This is accomplished by finding a set of tokens that should be valid and appending a help token
+        /// to that list, then re-parsing the list of tokens. This is not ideal - either we should have a direct way
+        /// of invoking help for a ParseResult.
```

Comment:
Missing the second part of either / or.  What's the other option?

Reply:
Pushed up a new commit with that, thanks! It summarizes future work we could do to get more System.CommandLine-ey with the way we invoke help.